utils/tracing/go/tracing_test.go

```diff
@@ -2,32 +2,61 @@ package tracing
 
 import (
 	"context"
+	"fmt"
 	"log"
```

Comment:
```suggestion
	"fmt"
	"log"
```
logrus logrus logrus logrus logrus logrus logrus logrus logrus logrus logrus logrus logrus logrus logrus logrus only